client/components/swipeable/style.scss

```diff
@@ -0,0 +1,29 @@
+@import '@wordpress/base-styles/mixins';
+.swipeable__container {
+	width: 100%;
+	height: 100%;
+	position: relative;
+	overflow: hidden;
+	list-style: none;
+	padding: 0;
+	touch-action: pan-y;
+}
+
+.swipeable__pages {
+	position: relative;
+	width: 100%;
+	height: 100%;
+	z-index: 1;
```

Comment:
I'd suggest being careful about hard-coded z-indexes. We try to make them relative throughout Calypso in order to make every element work properly across different areas of Calypso. They're managed in a centralized manner too, see https://github.com/Automattic/wp-calypso/blob/trunk/client/assets/stylesheets/shared/functions/_z-index.scss

Reply:
Good call. I remove the line.